apiserver/src/client/webclient.rs

```diff
@@ -0,0 +1,176 @@
+use super::error::{self, Result};
+use crate::{
+    constants::{
+        HEADER_BRUPOP_K8S_AUTH_TOKEN, HEADER_BRUPOP_NODE_NAME, HEADER_BRUPOP_NODE_UID,
+        NODE_RESOURCE_ENDPOINT,
+    },
+    CreateBottlerocketNodeRequest, DrainAndCordonBottlerocketNodeRequest,
+    UncordonBottlerocketNodeRequest, UpdateBottlerocketNodeRequest,
+};
+use models::{
+    constants::{APISERVER_SERVICE_NAME, APISERVER_SERVICE_PORT, NAMESPACE},
+    node::{BottlerocketNode, BottlerocketNodeSelector, BottlerocketNodeStatus},
+};
+
+use async_trait::async_trait;
+use snafu::ResultExt;
+use tokio::time::Duration;
+use tokio_retry::{
+    strategy::{jitter, ExponentialBackoff},
+    Retry,
+};
+use tracing::instrument;
+
+// The web client uses exponential backoff.
+// These values configure how long to delay between tries.
+const RETRY_BASE_DELAY: Duration = Duration::from_millis(100);
+const RETRY_MAX_DELAY: Duration = Duration::from_secs(10);
+const NUM_RETRIES: usize = 5usize;
```

Comment:
Is this a typo or a syntax? `5usize`

Reply:
It's syntax! Rust allows you to specify numeric literal types as part of literals to avoid having to add type annotations otherwise.

I think it's actually not necessary here, since we add a `usize` type annotation to the constant. I also think you can add `_` to these suffixes to make them more readable, e.g. `5_usize` would be one way to express it.